shared/surface_flux.F90

```diff
@@ -700,6 +724,15 @@ subroutine surface_flux_init
   read (fms_mpp_input_nml_file, surface_flux_nml, iostat=io)
   ierr = check_nml_error(io,'surface_flux_nml')
 
+  ! read rough_scheme from ocean_rough namelist
+  ! Note that we should not use the variable 'rough_scheme' directly from ocean_rough,
```

Comment:
I suggest change the comment to below to make the points more concise. There is a typo correction too. 

  ! Note that we should not use the value of 'rough_scheme' directly from ocean_rough,
  ! because the initialization of ocean_rough is later than the surface_flux_init.